.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 *.bak
 /target/
+/rls/
```

Comment:
How come RLS makes its directory in project root? For me it's always in `target/rls`